### ISSUE TITLE
onClick listener not working on MovingAverageTooltip, but working on OHLCTooltip

### ISSUE THREAD

**Issue by sulliwane** [rrag/react-stockcharts#157]
Hello,

I start playing a little bit with the first example (**[CandleStickChartWithDarkTheme](https://github.com/rrag/react-stockcharts/blob/master/docs/lib/charts/CandleStickChartWithDarkTheme.jsx)**) and I'd like to add a click listener on the moving average tooltip (component name: **[MovingAverageTooltip](https://github.com/rrag/react-stockcharts/blob/master/src/lib/tooltip/MovingAverageTooltip.jsx)**).

I cant' find why my "onClick" props is never called:
```javascript
<MovingAverageTooltip onClick={() => console.log('clicked')} origin={[-38, 10]} calculators={[ema20, ema50]}/>
```
Many thanks for your input on this!
cheers

**Comment by rrag** [comment]
It is a bug in 0.6 beta,

can you add this to your styles and let me know if that fixes it?

```
.react-stockcharts-moving-average-tooltip {
    pointer-events: all;
    cursor: pointer;
}
```

Once you confirm I can have it fixed.

**Comment by sulliwane** [comment]
that's it! I added it to [re-stock.scss](https://github.com/rrag/react-stockcharts/blob/master/docs/stylesheets/re-stock.scss), not sure if it's the best place but I can see my click now ! yay, Many thanks :)

**Comment by rrag** [comment]
It is fixed in `0.6-beta.5` you can remove that css from your stylesheet

**Comment by sulliwane** [comment]
Many thanks !

于 2016年10月28日 GMT+08:00 下午6:47:35, Ragu Ramaswamy <notifications@github.com> 写到:
